web/src/main/java/com/epam/rft/atsy/web/mapper/RuleValidationExceptionMapperImpl.java

```diff
@@ -1,25 +1,37 @@
 package com.epam.rft.atsy.web.mapper;
 
+import com.epam.rft.atsy.web.configuration.ExceptionMapperConfiguration;
 import org.springframework.stereotype.Component;
 import org.springframework.util.Assert;
 
 import java.util.Set;
 import javax.annotation.Resource;
 
+/**
```

Comment:
The javadocs are without line breaks after the summary sentence because of the current code formatting conventions that we applied to this project.